src/app/state/manager.rs

```diff
@@ -415,9 +415,11 @@ pub struct ManagerCreateSendTransactionState {
 
     vaults: Vec<ManagerSendInput>,
     outputs: Vec<ManagerSendOutput>,
-    feerate: u32,
+    feerate: Option<u32>,
     psbt: Option<(Psbt, u32)>,
     processing: bool,
+    warning_feerate: bool,
+    valid_feerate: bool,
```

Comment:
I think both can be removed now. I can find a way on the interface to trigger them

Reply:
I removed `warning_feerate`, `valid_feerate` is still there as it makes the button invalid when the feerate is empty